doc/wiki/source_code/Version_Control_System.md

```diff
@@ -1,83 +1,35 @@
-# Version Control System (VCS)
+# Revised Version Control System Page
 
-To help get everyone up to speed on the way we use version control, specifically Git
-on projects like MPICH, we've put together this Wiki page as a
-reference. Some of the content here may be remedial, but please go
-through all of it to make sure you're contributing code in the way that
-everyone else on the team expects. Useful resources on learning the basics of our
-current VCS, git, can be found below.
+To help get everyone up to speed on the way we use version control, specifically
+Git, on projects like MPICH. Please go through all of it to make sure you're
+contributing code in the way that everyone else on the team expects. For a 
+historical view on MPICH's version control systems, please visit the
+[Version Control History](Version_Control_History.md) page.
 
-- [Git Book](https://git-scm.com/book/en/v2)
-- [Git For Computer Scientists](http://eagain.net/articles/git-for-computer-scientists/)
-
-## Committing
-Below you can find information on what is required for creating a commit in MPICH.
-
-### Commit Content
```

Comment:
These commit guide lines are important. Why are they deleted?

Reply:
This content can now be found [here](https://github.com/travisk1154/mpich/blob/version_control_page_update/doc/wiki/source_code/Version_Control_System.md#create-a-commit)